node-fs-readfile/cat.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+const filename = process.argv;
+
+for (let index = 2; index < filename.length; index++) {
+  let read = '';
+  fs.readFile(filename[index], 'utf8', (err, data) => {
+    if (err) {
+      console.error(err);
+    }
+
+    read += data;
+    console.log(read);
+  });
+}
```

Comment:
This will read all the files the user specifies, but will not guarantee the order the files print out. Part of the challenge is to ensure the files always print in the same order the user input them in.